server.go

```diff
@@ -19,6 +20,32 @@ func fallback(w http.ResponseWriter, r *http.Request, reason string) {
 	http.Redirect(w, r, location, 302)
 }
 
+func getRedirect(txt []string, url string) (*Redirect, error) {
+	catch_alls := make(map[string]*Config)
+
+	for _, record := range txt {
+		config := Parse(record)
+		if strings.TrimSpace(config.From) == "" {
```

Comment:
Did you find that trimming is necessary here? The parser should only capture non-whitespace characters: https://github.com/holic/redirect.name/blob/master/parse.go#L12